Update docs on how to connect with Amazon OpenSearch Service via HTTP. (#8987)
opensearch-project/documentation-website: _clients/python-low-level.md

## Connecting to Amazon OpenSearch Service

> The following example illustrates connecting to Amazon OpenSearch Service via IAM credentials:

Reviewer: ```suggestion
The following example illustrates connecting to Amazon OpenSearch Service using IAM credentials:
```

    pool_maxsize = 20
)
```

> In case you're connecting via HTTP, with username and password, to Amazon OpenSearch Service, you could use the following:

Reviewer: ```suggestion
To connect to Amazon OpenSearch Service through HTTP with a username and password, use the following code:
```
